app/models/post.rb

```diff
@@ -1,5 +1,16 @@
 class Post < ApplicationRecord
+  after_save :update_posts_counter
   belongs_to :author, class_name: 'User'
   has_many :comments
   has_many :likes
+
+  def recent_comments(limit = 5)
+    comments.order(created_at: :desc).limit(limit)
+  end
+
+  private
+
+  def update_post_counter
```

Comment:
- [ ] There is little different between the method name in **line 2** and **line 12** respectively; I got `NoMethodError`. 👇 **Kindly use the same method name.**

<img width="727" alt="Untitled" src="https://github.com/ANTOINE1128/Blog-App1/assets/103386183/d4498d28-aeaf-4398-baa1-7aa43f9be024">